Source/WebCore/html/HTMLImageElement.cpp

```diff
@@ -508,6 +508,8 @@ void HTMLImageElement::setPictureElement(HTMLPictureElement* pictureElement)
     
 unsigned HTMLImageElement::width()
 {
+    ContentVisibilityForceLayoutScope scope(*const_cast<HTMLImageElement*>(this));
+
```

Comment:
In this method and all of the ones in `HTMLImageElement.cpp`, I think it would be good to create a block around the case where the renderer exists and to move the scope defined here into that block. That way we aren't doing extra work if the element has not been added to the document.